src/BeamAdapter/component/controller/InterventionalRadiologyController.inl

```diff
@@ -134,9 +134,29 @@ void InterventionalRadiologyController<DataTypes>::init()
     for(unsigned int i=0; i<m_instrumentsList.size(); i++)
         m_instrumentsList[i]->setControlled(true);
 
-    context->get(m_fixedConstraint);
-    if(m_fixedConstraint==nullptr)
-        msg_error()<<"No fixedConstraint found.";
+    if (!l_fixedConstraint)
+    {
+        typename FixedProjectiveConstraint<DataTypes>::SPtr fixedConstraint{};
+        context->get(fixedConstraint);
+        if (fixedConstraint)
+        {
+            l_fixedConstraint.set(fixedConstraint);
+        }
+        else
+        {
+            msg_error() << "No FixedConstraint found. One will be created on the spot but most likely it will not behave as attended.";
```

Comment:
```suggestion
            msg_error() << "No FixedProjectiveConstraint found. One will be created on the spot but most likely it will not behave as attended.";
```